hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/balanceof/BalanceOfCall.java

```diff
@@ -17,36 +17,31 @@
 package com.hedera.node.app.service.contract.impl.exec.systemcontracts.hts.balanceof;
```

Comment:
Successful refactor to `AbstractTokenViewCall` - this file is simpler/easier to read now